content/community/bug-bounty.mdx

```diff
@@ -27,7 +27,7 @@ You can also submit the bug reports on our Bug Bounty program on Immunefi.
 ### Bounty Scope
 The bug bounty will be applicable for the following repositories, sources and sites:
 
-* [https://github.com/PolymathNetwork/Polymesh/releases/tag/v3.0.0](https://github.com/PolymathNetwork/Polymesh/releases/tag/v3.0.0)
+* [https://github.com/PolymathNetwork/Polymesh/releases/tag/v5.0.0](https://github.com/PolymathNetwork/Polymesh/releases/tag/v5.0.0)
```

Comment:
We still don't have a v5.0.0 link but I have used it here so that we don't have to comeback on this again. I can link it to v5.0.0-rc1 if said so

Reply:
Can we link to the "releases" page instead so it will always be up to date instead of linking to a specific version.

https://github.com/PolymeshAssociation/Polymesh/releases

Reply:
This got updated to 5.0.2 in another PR. You can see the changes in merge branch [288c47a](https://github.com/PolymeshAssociation/technical-content/pull/180/commits/288c47a49ccb8bcdfd13a37f7e68691a5f525f6b)